src/modelldcatnotordf/informationmodel.py

```diff
@@ -5,18 +5,114 @@
 
 Refer to sub-class for typical usage examples.
 """
-from rdflib import Namespace
+from typing import List, Optional
 
+from datacatalogtordf import Resource, URI
```

Comment:
tror det er bedre om du oppretter egne versjoner av disse to

Reply:
Ok, @NilsOveTen , da skal jeg prøve å gjøre dette. :-)

Reply:
@stigbd Forstår det slik at dette er ditt forslag, jeg liker ikke at disse bibliotekene har dependencies mellom hverandre.

Om de skal bruke de samme objektene så vil jeg heller at vi har et biblitek spesifikkt for det, som disse bibliotekene importerer.

Reply:
@NilsOveTen klassen InformationModel er spesialiering av dcat:Resource. Det vil vere svært dumt å ikkje gjenbruke datacatalogtordf. I så fall må vi reimplementere 20+ properties (kopiere kode frå datacatalogtordf).
Eller misforstår eg poenget her?

Reply:
Ok, då må vi ta ein diskusjon på det. Men eg foreslår at vi flyttar den diskusjonen ut av denne PR. Det vil ha relativt store konsekvenser både for datacatalogtordf, concepttordf, oastodcat og brukarane av disse.

Reply:
Vi har en lintregel som sier "Black would make changes" og lint feiler om black ville gjort endringer.

Det er for sent å kjøre black ved deploy, hvis 'må' ha med automatisk kjøring av black så skal det kjøres i en precommit. Kode som deployes skal ikke vær forskjellig fra koden som pushes.